clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferReadGroupV1Metadata.scala

```diff
@@ -2,35 +2,37 @@ package org.broadinstitute.clio.transfer.model
 
 import java.time.OffsetDateTime
 
-case class TransferReadGroupV1Metadata(analysisType: Option[String],
-                                       baitIntervals: Option[String],
-                                       dataType: Option[String],
-                                       individualAlias: Option[String],
-                                       initiative: Option[String],
-                                       lcSet: Option[String],
-                                       libraryType: Option[String],
-                                       machineName: Option[String],
-                                       molecularBarcodeName: Option[String],
-                                       molecularBarcodeSequence: Option[String],
-                                       pairedRun: Option[Boolean],
-                                       productFamily: Option[String],
-                                       productName: Option[String],
-                                       productOrderId: Option[String],
-                                       productPartNumber: Option[String],
-                                       project: Option[String],
-                                       readStructure: Option[String],
-                                       researchProjectId: Option[String],
-                                       researchProjectName: Option[String],
-                                       rootSampleId: Option[String],
-                                       runDate: Option[OffsetDateTime],
-                                       runName: Option[String],
-                                       sampleAlias: Option[String],
-                                       sampleGender: Option[String],
-                                       sampleId: Option[String],
-                                       sampleLsid: Option[String],
-                                       sampleType: Option[String],
-                                       targetIntervals: Option[String],
-                                       notes: Option[String],
-                                       ubamMd5: Option[String],
-                                       ubamPath: Option[String],
-                                       ubamSize: Option[Long])
+case class TransferReadGroupV1Metadata(
```

Comment:
This snuck in when I split my commits, can revert if you want. Setting a default of `None` for all the optional fields in the transfer models makes building the models much cleaner in the integration tests, since you only need to specify the fields you care about.